src/typical.jl

```diff
@@ -90,16 +90,22 @@ function _replace(term::InteractionTerm, typicals::Dict)
 end
 
 _termsyms(t) = StatsModels.termsyms(t)
-_termsyms(t::FunctionTerm{Fo,Fa,Names}) where {Fo,Fa,Names} = Names
+
+@static if hasfield(FunctionTerm, :args)
+    # StatsModels 0.7
+    _termsyms(t::FunctionTerm) = StatsModels.termsyms.(t.args)
+else
+    # StatsModels 0.6
+    _termsyms(::FunctionTerm{Fo,Fa,Names}) where {Fo,Fa,Names} = Names
+end
 _symequal(t1::AbstractTerm, t2::AbstractTerm) = issetequal(_termsyms(t1), _termsyms(t2))
 
 _trmequal(t1::AbstractTerm, t2::AbstractTerm) = _symequal(t1, t2)
 _trmequal(t1::AbstractTerm, t2::FunctionTerm) = false
 _trmequal(t1::FunctionTerm, t2::AbstractTerm) = false
 function _trmequal(t1::FunctionTerm, t2::FunctionTerm)
     return t1.exorig == t2.exorig &&
-           _symequal(t1, t2) &&
-           t1.forig == t2.forig
+           _symequal(t1, t2)
```

Comment:
`forig` went away in 0.7 and it was already a very conservative check, so just dropped it